spec/index.html

```diff
@@ -977,8 +1130,93 @@ <h3>Overview</h3>
       <h3>Examples</h3>
 
       <p class="ednote">
-        Add some examples ranging from simple to complicated and resource consuming.
+        Add more examples ranging from simple to complicated and resource consuming.
       </p>
+
+      <aside id="ex-nd-shared-hashes"
+            class="example"
+            title="Shared hashes">
+        <p>This example revisits <a href="#ex-shared-hashes"></a>
+          to illustrate the operation of the
+          <a href="#hash-nd-quads-algorithm">Hash N-Degree Quads Algorithm</a>
+          on the blank nodes (`_:b0` and `_:b1`) which resulted in a shared result from the
+          <a href="#hash-1d-quads-algorithm">Hash First Degree Quads Algorithm</a>.</p>
+
+        <figure id="fig-nd-shared-hashes" style="text-align:center">
+          <!-- Source for this file is at https://docs.google.com/drawings/d/1HWpz8S-PNaWi02-utF62HAMd_0b-AF_tHNSRRUQILbU/edit?usp=sharing -->
+          <object data="shared-hashes.svg" style="width: 75%" type="image/svg+xml" aria-describedby="fig-nd-shared-hashes-alt">
+            <p id="fig-nd-shared-hashes-alt">
+              The image represents the graph described in
+              <a href="#ex-nd-shared-hashes-input">
+                the following code block
+              </a>.</p>
+          </object>
+          <figcaption>An illustration of a graph resulting in shared hashes.<br/>
+            Image available in
+            <a href="shared-hashes.svg" title="SVG image of a graph resulting in shared hashes">
+              <abbr title="Scalable Vector Graphics">SVG</abbr>
+            </a>.</figcaption>
+        </figure>
+          
+        <pre id="ex-nd-shared-hashes-input"
+             data-transform="updateExample"
+             title="Graph with blank nodes resulting in shared hashes">
+          <!--
+            :p :q ****_:b0**** .
+            :p :q ****_:b1**** .
+            ****_:b0**** :p _:b2 .
+            ****_:b1**** :p _:b3 .
+            _:b2 :r _:b3 .
+          -->
+        </pre>
+
+        <p>The algorithm will be called twice, with each blank node
+          (`_:b0`, `_:b1`) which did not result in a unique hash
```

Comment:
I think the algorithm will actually be called 4 times (another time each for `_:b2` and `_:b3`) here. I understand that we're looking to continue the example from earlier so maybe this is meant to be further edited in some way -- just leaving a note here.

Reply:
This example should have been left out entirely, but was added back accidentally. Examples for Hash N-Degree Quads need more work, which I was going to do in a different PR later.